components/dialog/dialog-fullscreen.js

```diff
@@ -53,17 +53,18 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 				}
 
 				.d2l-dialog-content {
-					display: grid;
+					/* 1 rem padding plus 1px transparent border */
+					padding-bottom: 21px;
+					padding-top: 1rem;
 				}
 
 				.d2l-dialog-content > div {
 					/* required to properly calculate preferred height when there are bottom
 					margins at the end of the slotted content */
 					border-bottom: 1px solid transparent;
 					box-sizing: border-box;
+					display: contents;
```

Comment:
Removing display: grid and adding display: contents here allowed the content width to be fixed. It seems like everything is working fine with this, but I'll need to do some regression testing within the lms to make sure.

Reply:
Something worrying from the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/display):

> Due to a bug in browsers this will currently remove the element from the accessibility tree — screen readers will not look at what's inside.

The [cross-browser support](https://caniuse.com/css-display-contents) is also a little sketchy... specifically legacy-Edge doesn't support it and there are some scary yellow yellow boxes for Safari and relatively recent Chrome/Firefox.

Reply:
That's not great 😢 I'll keep looking for another solution. I'm having trouble getting the content to stay a fixed width, and keeping the top/bottom padding.